### PR TITLE
B1

### PR DESCRIPTION
**Pull request by Arnab-Developer**

### CODE REVIEW

File: f2.md [hunk 1]
```diff
@@ -1 +1 @@
-
+Init text
```
Comment (arnab-jit1) [inline f2.md:1]: Give stop

File: f1.md [hunk 2]
```diff
@@ -1 +1 @@
-
+Init text
```
Comment (arnab-jit1) [inline f1.md:1]: Give stop

**Review by arnab-jit1 — CHANGES_REQUESTED** [review]

File: f2.md (hunk 1 above)
Comment (Arnab-Developer) [reply f2.md]: Done

File: f1.md (hunk 2 above)
Comment (Arnab-Developer) [reply f1.md]: Done

File: f1.md [hunk 3]
```diff
@@ -1 +1 @@
-
+Init text..
```
Comment (arnab-jit1) [inline f1.md:1]: Use one stop

**Review by arnab-jit1 — CHANGES_REQUESTED** [review]

File: f1.md (hunk 3 above)
Comment (Arnab-Developer) [reply f1.md]: Done

**Review by arnab-jit1 — APPROVED** [review]

**Pull request merged**